### PR TITLE
Some formulation options

### PR DESCRIPTION
**Pull request by paveltomin**
Few experimental options for flow solver:

`useTotalMassEquation` - enabled by default, when disabled - no total mass equation will be formed

`useSimpleAccumulation` - disabled by default, when enabled - the accumulation term will be assembled using simple form: `PV * rho_c`, mathematically equivalent to the current version, numerically might be slightly different. Simplifies accumulation linearization.

`minCompDens` - default value 1e-10 (same as before), allows experiments with minimum component ammount.

Also added zero diagonal check in accumulation linearization.

Packed flags into bit flags to pass into kernels.

Changed `co2_flux_3d.xml` to use new flags.

Rebaseline PR: https://github.com/GEOS-DEV/integratedTests/pull/56

### CODE REVIEW

**Comment by paveltomin** [comment]
review anyone?

**Comment by paveltomin** [comment]
review anyone? all changes are flag-protected and rebaseline required only because of the new flags introduced in xml

**Review by klevzoff — APPROVED** [review]
Nice and clean implementation! Sorry this took so long to review. Just a couple minor comments on `BitFlags`.

File: src/coreComponents/physicsSolvers/fluidFlow/IsothermalCompositionalMultiphaseBaseKernels.hpp [hunk 1]
```diff
@@ -720,6 +743,9 @@ class ElementBasedAssemblyKernel
         real64 const dPhaseCompAmount_dP = dPhaseAmount_dP * phaseCompFrac[ip][ic]
                                            + phaseAmount * dPhaseCompFrac[ip][ic][Deriv::dP];
 
+        GEOS_ASSERT_MSG( ic == 0 && LvArray::math::abs( dPhaseCompAmount_dP ) < minDensForDivision,
+                         "Zero diagonal in Jacobian" );
```
Comment (paveltomin) [inline src/coreComponents/physicsSolvers/fluidFlow/IsothermalCompositionalMultiphaseBaseKernels.hpp:747]: @klevzoff is it allowed or bad?

Comment (klevzoff) [reply src/coreComponents/physicsSolvers/fluidFlow/IsothermalCompositionalMultiphaseBaseKernels.hpp:747]: Having an assertion in kernel is fine, keep in mind it is only enabled in debug builds. You want to negate the condition here though, `GEOS_ASSERT` works opposite of `GEOS_ERROR_IF`.

Comment (paveltomin) [reply src/coreComponents/physicsSolvers/fluidFlow/IsothermalCompositionalMultiphaseBaseKernels.hpp]: Just corrected, thanks!

**Pull request merged**